sympy/solvers/ode/systems.py

```diff
@@ -169,7 +171,10 @@ def linear_ode_to_matrix(eqs, funcs, t, order):
         except NonlinearError:
             raise ODENonlinearError("The system of ODEs is nonlinear.")
 
+        Ai = Ai.applyfunc(expand)
```

Comment:
Maybe this should just be `expand_mul`. What is this needed for?

Reply:
Ok, I will update this, it's not needed anymore actually.